src/pcapi/core/offers/api.py

```diff
@@ -410,6 +409,8 @@ def upsert_stocks(
         if offer.validation == OfferValidationStatus.PENDING or offer.validation == OfferValidationStatus.REJECTED:
             offer.isActive = False
         repository.save(offer)
+        if offer.validation == OfferValidationStatus.APPROVED and user:
```

Comment:
```suggestion
        if offer.validation == OfferValidationStatus.APPROVED:
```
C'est obligé d'avoir user non ?